src/kvstore/wal/FileBasedWal.cpp

```diff
@@ -293,7 +293,8 @@ void FileBasedWal::closeCurrFile() {
     timebuf.modtime = currInfo_->mtime();
     timebuf.actime = currInfo_->mtime();
     VLOG(1) << "Close cur file " << currInfo_->path() << ", mtime: " << currInfo_->mtime();
-    CHECK_EQ(utime(currInfo_->path(), &timebuf), 0);
+    CHECK_EQ(utime(currInfo_->path(), &timebuf), 0) << "file: " << currInfo_->path()
```

Comment:
There is no need to CHECK_EQ, if utime return fails, then printing a log is enough.

Reply:
Thanks.